include/openspace/properties/property.h

```diff
@@ -62,13 +62,15 @@ class Property {
 public:
     /**
     * The visibility classes for Property%s. The classes are strictly ordered as
-    * All > Developer > User > Hidden
+    * Hidden > Developer > AdvancedUser > User > NoviceUser > Always
     */
     enum class Visibility {
-        Hidden = 3, ///< Never visible
-        Developer = 2, ///< Visible in Developer mode
-        User = 1, ///< Visible in User mode
-        All = 0,  ///< Visible for all types, no matter what
+        Hidden = 5, ///< Never visible
```

Comment:
Do we really need all of these categories? Might be tricky/annoying to maintain

We should all have a discussion as a team about this when we have the chance. (Talked about it briefly during the dev meeting on May 23)

@alexanderbock

Reply:
Yea, let's have a talk about it, but I think we do:
 - `Hidden` and `Always` we will need for sure as sentinel values
 - `Developer` and `User` also seems a logical choice to me.
 -  `AdvancedUser` would be the types of properties that we would want knowledeable users to be able to interact with (for example the atmosphere scattering parameters, but others too)
 - `NoviceUser` the type of properties that would result in a "simplified ui" that could work as a first version for #202

Reply:
and the "Advanced User" / "User" / "Novice User" has a nice symmetry to it